### PR TITLE
Ticket 95: Recommend limit for get-entries response size.

### PR DESCRIPTION
**Pull request by eranmes**

### CODE REVIEW

File: rfc6962-bis.xml [hunk 1]
```diff
@@ -1130,7 +1130,7 @@ treating unrecognized leaves as opaque input to the tree.
         </t>
         <t>
           Logs MAY restrict the number of entries that can be retrieved per
-<spanx style="verb">get-entries</spanx> request.  If a client requests more
+<spanx style="verb">get-entries</spanx> request (the limit of 1,000 entries is suggested).  If a client requests more
```
Comment (benlaurie) [inline rfc6962-bis.xml]: Why are we suggesting a limit? Also "_a_ limit..."

**Comment by benlaurie** [comment]
My position on ticket 95 hasn't changed. I don't see any use in this. Instead propose we close ticket 95.

**Comment by robstradling** [comment]
For crt.sh's, I currently fetch (up to) 1,024 entries per get-entries request.  I arrived at this strategy by trial and error.  It would be nice to have more determinism here, but simply suggesting a limit in the doc wouldn't actually help with that, because I'd still have to figure out what the logs are actually doing by trial and error.  So I agree with Ben: let's close ticket 95 as a WONTFIX.

**Comment by eranmes** [comment]
Abandoning this PR as suggested by Ben and Rob.

**Pull request closed without merging**